README.md

```diff
@@ -16,6 +16,20 @@ The Chainlink Testing Framework is a blockchain development framework written in
 
 If you're looking to implement a new chain integration for the testing framework, head over to the [blockchain](./blockchain/) directory for more info.
 
+## Developing
+
+In order to work with CTF, you'll need a few dependencies in place. First make sure to have [node](https://nodejs.org/en/download) installed. Then run the below:
+
+```sh
+make install
+```
+
+We also recommend using our handy githooks.
+
+```sh
+git config core.hooksPath .githooks
+```
```

Comment:
Do you want to instead just replace the `pre-commit install` that exists in the nix file and the `make install`, otherwise they will get a different pattern. If they don't run this command then they get both the pre-commit and pre-push running on commit

Reply:
Yes, I'm unfamiliar with nix though. I'm assuming the answer is the `pre-commit-config.yaml`?

Reply:
Just change in it nix here: https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/shell.nix#L60